docs/pages/billing/manage.mdx

```diff
@@ -0,0 +1,262 @@
+---
+title: Manage plans and billing
+description: Learn how to update, downgrade, or cancel your Expo account's plans and manage billing details.
+---
+
+import { Tabs, Tab } from '~/ui/components/Tabs';
+import ImageSpotlight from '~/components/plugins/ImageSpotlight';
+
+**Billing** in the Expo dashboard provides information about your account's currently subscribed plan and monthly usage. It also allows you to manage your plan and billing details.
+
+This guide explains how to manage your account's plans and billing information.
+
+## Manage plans
+
+### View the current plan
+
+<Tabs>
+
+<Tab label="For Organization account">
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu under **Organization settings**.
+- Under **Plan**, you can see the current plan for your account.
+
+For example, an Organization account is subscribed to the Production plan below:
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the current plan for an Organization account."
+  src="/static/images/billing/management/billing-01.png"
+/>
+
+</Tab>
+
+<Tab label="For Personal account">
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu under **Account settings**.
+- Under **Plan**, you can see the current plan for your account.
+
+For example, a Personal account is subscribed to the Production plan below:
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the current plan for a Personal account."
+  src="/static/images/billing/management/billing-02.png"
+/>
+
+</Tab>
+
+</Tabs>
+
+### Upgrade to a new plan
+
+To upgrade to a different plan:
+
+- Click [**Billing**](https://expo.dev/settings/billing) from the navigation menu in Expo dashboard.
+- Under **Plan**, click **Change Plan** if you are already on a paid plan. If you are on the Free plan, click **Subscribe**. It opens the **Cart** page.
+- Under **Choose a plan**, you can see a list of all available plans. Choose the next plan you want to upgrade and click the **Continue to Review** button at the bottom of this page.
```

Comment:
Replace "Choose the next plan you want to upgrade" with "Choose the plan you want to upgrade to"